less/less.d.ts

```diff
@@ -72,6 +77,8 @@ declare namespace Less {
 }
 
 interface LessStatic {
+    options: Less.StaticOptions;
```

Comment:
Theres some weirdness with the way you configure less. As you pointed out the docs say that you have to declare a global variable called less prior to less being loaded. Once less has been loaded that options object is then stored within lessStatic